File: app/src/main/java/com/hzw/android/richman/utils/LogUtil.java
package com.hzw.android.richman.utils;

import android.util.Log;

/**
 * class LogUiti
 *
 * @author CrazyDragon
 * description
 * note
 * create date 2022/2/9
 */
public class LogUtil {

    final static String TAG = "日志";

    public static void print(String msg) {
        Log.e(TAG, msg);
    }

    public static void print(int msg) {
        Log.e(TAG, String.valueOf(msg));
    }
} 

File: app/src/main/java/com/hzw/android/richman/utils/LogUtil.kt
package com.hzw.android.richman.utils

import android.util.Log

/**
 * class LogUtil
 *
 * @author CrazyDragon
 * description 日志工具
 * note
 * create date 2022/2/9
 */
object LogUtil {

    private const val TAG = "日志"

    fun print(msg: String) {
        Log.e(TAG, msg)
    }

    fun print(msg: Int) {
        Log.e(TAG, msg.toString())
    }
}